### ISSUE TITLE
Add zoom feature

### ISSUE THREAD

**Issue by ranshaa05** [pyppeteer/pyppeteer#322]
Definitely not an urgent feature, but I think that having the option to zoom in on a page could come in handy.

**Comment by Mattwmaster58** [comment]
The aim of the project is to replicate puppeteer's API. We're quite behind on that, but if puppeteer adds this we may also add it sometime in the (maybe perpetual) future

**Comment by JavaScriptDude** [comment]
This already exists, its just not called zoom.

To use, pass `'scale' = <factor>` in page options dict to `page.pdf(<page_opts>)`.

Eg: `page.pdf({'path': '<out_path>', 'scale': 1.5})` and this will scale the document 1.5 times.

Note - It appears that this is not implemented in the [screenshot API](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-captureScreenshot).

**Comment by Nordgaren** [comment]
> This already exists, its just not called zoom.
> 
> To use, pass `'scale' = True` in page options dict to `page.pdf(<page_opts>)` or `page.screenshot(<page_opts>)`.
> 
> Eg: `page.pdf({'path': '<out_path>', 'scale': 1.5})` and this will scale the document 1.5 times.

This doesn't seem to work on page.screenshot :(

```
page.screenshot({'path': 'C:\\Users\\Nord\\source\\repos\\Python\\Print Web\\screenScaled.png', 'scale': 3.0, 'fullPage': True})
```
the command I am trying to run, for reference.

Am I doing something wrong?

**Comment by Mattwmaster58** [comment]
Likely not implemented. I would check the source to make sure

**Comment by JavaScriptDude** [comment]
You are correct, the the screenshot API is not implemented in the [Page.captureScreenshot Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-captureScreenshot).

I've updated my comment above to reflect this.

**Comment by Nordgaren** [comment]
ty both!

I think I can implement it by just adding scale to the dictionary and setting it, here!

I will give this a shot and then send a PR!

```py
if self._viewport is not None:
    mobile = self._viewport.get('isMobile', False)
    deviceScaleFactor = self._viewport.get('deviceScaleFactor', 1)
    landscape = self._viewport.get('isLandscape', False)
else:
    mobile = False
    deviceScaleFactor = 1
    landscape = False
```

Edit: PR Sent. Hope it's up to par!
